recipes/lua-5.1/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: lua
+  version: 5.1.5
```

Comment:
As in: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L1-L5

Except change 1.0 to 5.1.5?